files/en-us/web/accessibility/understanding_colors_and_luminance/index.html

```diff
@@ -2,289 +2,470 @@
+<h2 id="Modeling_Light_Color_and_Vision">Modeling Light Color and Vision</h2>
+
+<p>To help us in our work, be it content design or determining accessibility, or calibrating a monitor, we need a way to model and predict the behavior and perception of light and color. These models are built with basic math, and are specific to the given task. A colorimetric model is one that is involved in measuring light, an appearance model is designed to predict how we will perceive the light as measured.</p>
+
+
+
+<h3 id="Calculating_Relative_Luminance">Calculating Relative Luminance</h3>
+
+<p>The IEC standard for calculating relative luminance (Y) from an sRGB color is as follows:</p>
+
+<h4>STEP ONE (8 bit integer to float)</h4>
+
+<p>Convert 8 bit sRGB values to float (0.0 to 1.0) by dividing by 255.0:</p>
+
+<p>&nbsp;&nbsp; <strong>R´<sub>float</sub> = R´<sub>8bit</sub> / 255.0 &nbsp;&nbsp; &nbsp;&nbsp; G´<sub>float</sub> = G´<sub>8bit</sub> / 255.0 &nbsp;&nbsp; &nbsp;&nbsp; B´<sub>float</sub> = B´<sub>8bit</sub> / 255.0</strong></p>
+
+<p>If your sRGB values are 16 bit then convert to decimal by dividing by 65535.</p>
+
+<h4>STEP TWO (Linearize)</h4>
+<p>Linearize the data to remove the encoded transfer curve.</p>
+
+<p>&nbsp;&nbsp; <strong>R<sub>lin</sub> = sRGBtoLin(R´<sub>float</sub>) &nbsp;&nbsp;   G<sub>lin</sub> = sRGBtoLin(G´<sub>float</sub>)   &nbsp;&nbsp; B<sub>lin</sub> = sRGBtoLin(B´<sub>float</sub>) </strong></p>
+
+<p>Using the function shown below:</p>
+
+<pre class="brush: javascript" style="">function sRGBtoLin(colorChannel) {
+		  // Send this function a decimal sRGB gamma encoded color channel
+		  // between 0.0 and 1.0, and it returns a linearized value.
+		if ( colorChannel &lt;= 0.04045 ) {
+			return colorChannel / 12.92;
+		} else {
+			return Math.pow((( colorChannel + 0.055)/1.055),2.4);
+		}
+	}</pre>
+
+<div class="note notecard">
+<h4>IMPORTANT NOTE</h4>
+<p><em>Those familiar with the WCAG 2.x contrast math may notice that the above code uses the threshold value of 0.04045. This is the official IEC standard. The WCAG 2.x guidelines were drafted citing an obsolete value. While the WCAG 2.x version is technically incorrect, with 8-bit color there is no functional difference for the purpose of WCAG 2.x contrast. For the record, the WCAG 2.x value is 0.03928 </em></p></div>
+
+<h4>STEP THREE (Spectrally Weighted Luminance)</h4>
+
+<p>The normal human eye has three types of cones that are sensitive to red, green, and blue light. But our spectral sensitivity is not uniform, as we are most sensitive to green (555 nm), and blue is a distant last place. Luminance is spectrally weighted to reflect this using the following coefficients for sRGB:</p>
+
+<p>&nbsp;&nbsp; <strong>Y = R<sub>lin</sub> * 0.2126 + G<sub>lin</sub> * 0.7152 + B<sub>lin</sub> * 0.0722</strong></p>
+
+<p>Multiply each linearized color channel by their coefficient and sum them all together to find Y, Luminance.</p>
+
+<hr>
+<h4>ALTERNATE (sRGB to Y, simple)</h4>
```

Comment:
```suggestion
<h4>Alternate: sRGB to Y, simple</h4>
```

This `hr` feels out of place given how it is previously used.